Misc/NEWS.d/next/Library/2018-06-07-18-55-35.bpo-32493.1Bte62.rst

```diff
@@ -0,0 +1 @@
+correct logic in configure.ac so that HAVE_UUID_ENC_BE is not always defined
```

Comment:
Please start the sentence from the title case, and add a period at the end.

Reply:
The period at the end - I understand, but now clue what you mean about starting the sentence from the title case.

Happy to change the text - but is it really so different from the others? I am not the only one to have not ended the string with a period.

Now it just takes another 24 hours before anything happens - you answer, I find time to change it, the PR gets updated, run through the verification, and then maybe it gets merged - and this breaks - for AIX - what the prior PR was fixing.

Ned - I may have misunderstood re: master and 3.7, but I thought, in emails with victor - a quick PR based on 3.7 was okay (if not preferred). This is so I can back on another PR for uuid and AIX that won't work until this merged.

Reply:
By “title case”, I think he means start the sentence with a capital letter: “correct logic” becomes “Correct logic”